src/main/resources/assets/bettergrass/lang/zh_tw.json

```diff
@@ -11,12 +11,12 @@
   "bettergrass.crimsonNylium.desc": "也會連接緋紅菌絲石！",
   "bettergrass.warpedNylium.desc": "也會連接扭曲菌絲石！",
   "bettergrass.moreBlocks.desc": "應該在斜坡上連接的類似草地的方塊。",
-  "bettergrass.betterSnow": "Better Snow",
-  "bettergrass.betterSnowMode": "Better Snow Mode",
-  "bettergrass.betterSnowMode.desc": "§4Off §r- No snow layer will be generated.\n§3OptiFine-like §r- Shows a snow layer beneath specific blocks that have at-least one snow layer or snow block near them.\n§dLambdaBetterGrass-like §r- Shows a snow layer beneath specific blocks that have at-least \"two\" snow layers or snow blocks near them.",
-  "bettergrass.betterSnowMode.optifine": "OptiFine-like",
-  "bettergrass.betterSnowMode.lambda": "LambdaBetterGrass-like",
-  "bettergrass.excludedTags": "Excluded Tags",
-  "bettergrass.excludedBlocks": "Excluded Blocks",
+  "bettergrass.betterSnow": "更好的雪地",
+  "bettergrass.betterSnowMode": "更好的雪地",
```

Comment:
Can you change this from `Better Snow` --> `Better Snow Mode`?

Reply:
You mean translation or the string key?

Reply:
translation